client/state/ui/olark/actions.js

```diff
@@ -7,7 +7,8 @@ import {
 	OLARK_REQUEST,
 	OLARK_TIMEOUT,
 	OLARK_OPERATORS_AVAILABLE,
-	OLARK_OPERATORS_AWAY
+	OLARK_OPERATORS_AWAY,
+	OLARK_SET_AVAILABILITY,
```

Comment:
The dangling comma is allowed so I've just become accustomed to using it all the time now.